packages/eslint-plugin/src/rules/import-order.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * @fileoverview Disallows the use of require statements except in import statements.
```

Comment:
Sorry we've changed to using all-contributors now and removed all other occurrences of these file-level author comments, please could you remove this